setup.cfg

```diff
@@ -1,3 +1,4 @@
 [flake8]
 ignore = E265
 exclude=webapp-django/crashstats/*/migrations/*,docs/conf.py,socorro/unittest/testlib/*
+max-line-length = 100
```

Comment:
Plus the times I'd editing in an ssh window in jed and not using my fancy Atom.

But if the team prefers 100 I'm happy to change.